up.go

```diff
@@ -63,3 +63,24 @@ func UpByOne(db *sql.DB, dir string) error {
 
 	return nil
 }
+
+// UpAll migrates up all pending migrations.
+func UpAll(db *sql.DB, dir string) error {
```

Comment:
I'd expect this function to call `Up()` until it hits error or last migration.

Reply:
Really I don't understand what do you mean by that. Maybe my other comments will clarify why I did this function like that. Or could you please give me some explanation.